index.html

```diff
@@ -242,6 +242,46 @@ <h1>About me</h1>
             </ul>
         </div>
     </section>
+    <!--Contact-form-section-->
+    <section class="contact-form">
+        <form action="https://formspree.io/f/xayzobow" method="post" >
+            <h1 id="contact-header">I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</h1>
+        <fieldset>
```

Comment:
- [ ] Please update your design to match the original design  the colors, outline for inputs and margin and colors

- Your Design 👇 

![image](https://user-images.githubusercontent.com/75126481/220440838-66ac24c3-8877-4718-9241-7634ec7b7dd5.png)

- Original Design 👇 

![image](https://user-images.githubusercontent.com/75126481/220440984-20503fd4-9237-4097-84a6-5fc833c25539.png)